### DISCUSSION TITLE
ksads youth 3-year follow-up on release 5.1?

### DISCUSSION THREAD

**Discussion by njokada** [now-i-know-my-abcd/docs#63] [Where in the World is…? (Help)] [answered]
hi there,

i am looking for ksads dx on release 5.1 for the 3-year follow-up. it seems as though the variables (prefix ksads2_) which correspond to the ksads 2.0 (which the participants received at the 3-year timepoint) are mostly NA?

was wondering if anyone had any insight on this.

thank you!!
nana

**Comment by clarefmccann** [comment, marked as answer]
Hi Nana, 

Thanks for posting! Looking at the ABCD protocols (https://abcdstudy.org/scientists/protocols/), it seems that they only collected KSADS modules for suicide and substance use-related behavior. I believe they will collect a full KSADS in the fourth-year follow-up. 

Clare

**Reply by njokada** [reply]
Thank you so much!
